include/ck/tensor_operation/gpu/element/unary_element_wise_operation.hpp

```diff
@@ -0,0 +1,87 @@
+#pragma once
+#include "data_type.hpp"
+#include "math_v2.hpp"
+
+namespace ck {
+namespace tensor_operation {
+namespace element_wise {
+
+struct PassThrough
+{
+    __host__ __device__ void operator()(float& y, const float& x) const { y = x; }
```

Comment:
How about use is_same to replace overloadding?

Reply:
Yes.   Use static_assert  in [1a444](https://github.com/ROCmSoftwarePlatform/composable_kernel/pull/274/commits/61d01e71055f76b34b5a5e3b9f064f6cfbf1a4ee)